forem/ForemWebView-android
# Fix #26: Implemented WebViewStatus to get the current status of webview

## What type of PR is this? (check all applicable)


## Description

Fix #26: Implemented WebViewStatus to get the current status of webview

- LOADING
- SUCCESSFUL
- ERROR

## Related Tickets & Documents



- Related Issue #26
- Closes #26

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

- Run this branch
- Once one of the forem is visible, click at bottom-right corner name of forem
- Notice the loading indicator in between.

### UI changes
_If your PR includes UI changes, please share before/after screenshots or videos for comparison._

- Loading is displayed whenever there is change in Forem.

<img src="https://user-images.githubusercontent.com/9396084/181616107-82da7690-3cbb-447c-a597-d55f477bdfe3.png" width="250" />

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested._
None.

## Added/updated tests?

- [x] No, and this is why: Currently all Espresso tests are failing even on branches there they were written and passed successfully indicating there is some issue with android studio mostly. Will see how these tests work in Robolectric.
